src/explain.rkt

```diff
@@ -5,7 +5,7 @@
          "ground-truth.rkt" "syntax/sugar.rkt" "alternative.rkt" "programs.rkt"
          "float.rkt" "config.rkt")
 
-(provide explain)
+(provide explain compile-expr predict-errors gen-expls-table)
```

Comment:
Why do all of these methods need to be provided to callers? I don't think I see anyone calling them?

Reply:
This is my fault. I forgot I return the explanation table from explain. I think you don't need to provide these functions and I don't think gen-expls-table is needed, we can remove the whole function.